src/sentry/static/sentry/app/components/group/sentryAppExternalIssueForm.tsx

```diff
@@ -156,6 +169,10 @@ class SentryAppExternalIssueForm extends React.Component {
 
         {requiredFields.map(field => {
           field.choices = field.choices || [];
+          field.inline = false;
```

Comment:
Hmmmm, I would be careful with this, you're mutating `this.props.config` and can cause problems for other components that might use `this.props.config`.

Reply:
@billyvg Good point. The mutation was already there but definitely should fix that.